core/modules/wbase/SendChannel.h

```diff
@@ -42,16 +42,17 @@ namespace wbase {
 /// accepts only one call to send bytes, unless the sendStream call is used.
 class SendChannel {
 public:
-    typedef util::VoidCallable<void> ReleaseFunc;
-    typedef std::shared_ptr<ReleaseFunc> ReleaseFuncPtr;
-    typedef long long Size;
+    using Ptr = std::shared_ptr<SendChannel>;
+    struct ReleaseFunc {
```

Comment:
I'll need to look into it more, but it can be tough to track back std::function or util::VoidCallablke<void> back to their source. It's pretty easy to find all the children of ReleaseFunc, see what they do, fix all of them, etc. I think it makes the code easier to read and we don't need generic capabilities here.